intern-book-app/src/app/pages/list/list.component.ts

```diff
@@ -9,6 +10,25 @@ import { Book } from 'src/app/types/book';
 export class ListComponent implements OnInit {
   title = '読んだ本';
 
+  book: Book = {
+    name: '',
+    detail: '',
+    evaluation: 0,
+  };
+
+  formReset() {
+    //各パラメータに初期値を代入
+    (this.book.name = ''), (this.book.detail = ''), (this.book.evaluation = 0);
+  }
+
+  addBook() {
+    this.bookList.push({
+      name: this.book.name,
+      detail: this.book.detail,
+      evaluation: this.book.evaluation,
+    });
+  }
```

Comment:
`formReset`は`bookList`に追加された後に実行されると嬉しいね！
なので、buttonに複数`(click)`は付けずに、`push`後に実行してあげましょう〜
```suggestion
  addBook() {
    this.bookList.push({
      name: this.book.name,
      detail: this.book.detail,
      evaluation: this.book.evaluation,
    });
    this.formReset();
  }
```